options.c

```diff
@@ -3947,13 +3941,28 @@ struct fio_option fio_options[FIO_MAX_OPTS] = {
 		.group	= FIO_OPT_G_PROCESS,
 	},
 	{
-		.name	= "exit_every_group",
-		.lname	= "Exit-every group on terminate",
-		.type	= FIO_OPT_STR_SET,
-		.cb	= str_exit_every_group_cb,
-		.help	= "Terminate all jobs of all groups when one jobs exits",
+		.name	= "exit_what",
+		.lname	= "Exit-all on terminate",
```

Comment:
Should this be updated?

Reply:
Yes, the documentation is also not ready yet. I think with the current implementaiton of exit_what I also need to edit cconv.c, right? I'm planning also to add an example.

Anyways I want to start updating documentation when the code is nailed down, otherwise it's too much of a moving goal post …

Reply:
You need cconv additions if you make it a job option, instead of a global option. Which is recommended, global options are discouraged and honestly should not exist. They don't work over the network, for example.